mlflow/gateway/providers/bedrock.py

```diff
@@ -56,16 +66,22 @@ def completions_to_model(cls, payload, config):
     @classmethod
     def model_to_completions(cls, resp, config):
         return completions.ResponsePayload(
-            **{
-                "metadata": {
-                    "model": config.model.name,
-                    "route_type": config.route_type,
-                },
-                "candidates": [
-                    {"text": candidate.get("outputText"), "metadata": {}}
-                    for candidate in resp.get("results", [])
-                ],
-            }
+            created=int(time.time()),
+            object="text_completion",
+            model=config.model.name,
```

Comment:
No temperature scaling to be done for AI21Labs. Range is -Inf to -Inf for AI21. We can stick with 0-2 restriction for now (it's currently 0-1) and get feedback.